README.md

```diff
@@ -95,6 +95,8 @@ a parameter type.
 | `useForSnippets` / `use_for_snippets` | Defaults to `true`. That means this parameter type will be used to generate snippets for undefined steps. If the `regexp` frequently matches text you don't intend to be used as arguments, disable its use for snippets with `false`. |
 | `preferForRegexpMatch` / `prefer_for_regexp_match` | Defaults to `false`. Set to `true` if you have step definitions that use regular expressions, and you want this parameter type to take precedence over others during a match. |
 
+*Note: Currently the parameter name cannot be customized, so the custom parameters can only be used with the type name, e.g. `{Color}`.*
```

Comment:
This doesn't sound accurate. I know it does work in Cucumber-JVM. Perhaps this should be a bug report for a specific Cucumber implementation?

Reply:
Moved to #277